### PR TITLE
Sam Nayrouz-JS Basics HW

### PR DESCRIPTION
**Pull request by snayrouz**

### CODE REVIEW

File: lib/times_table.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function timesTable(numRows) {
+  let rows = '';
+
+  for (let i = 1; i <= numRows; i++) {
+    let row = '';
+    for (var j = 1; j <= numRows; j++) {
+      row += `${(i * j)}\t`;
+    }
+    rows += `${row}\n`;
```
Comment (laurenfazah) [inline lib/times_table.js]: Is there an Array.prototype message you could research that could clean this up?

Comment (snayrouz) [reply lib/times_table.js]: Ya most likely. Let me look at it when I have time today.

Comment (laurenfazah) [reply lib/times_table.js]: No problem! Just pushing you to think that way, at least for the future

**Comment by laurenfazah** [comment]
These tests and solutions look great, Sam! Like I said in that inline comment, I'd now push yourself to leverage more Array.prototype methods, especially when you start seeing lots of for loops in your code.

**Pull request closed without merging**